README.md

```diff
@@ -185,6 +185,14 @@ We deploy from CircleCI.
 
 When CircleCI tries to deploy two PRs back-to-back, one of them can fail. In this case, you can restart the failed deploy process by clicking the "Rerun Workflow" button.
 
+## Additional documentation
+
+For more technical documentation see the [docs](https://github.com/usdoj-crt/crt-portal/tree/develop/docs)
```

Comment:
Ooooh, adding this is a great catch! I really appreciate your documentation thoroughness @LindsayYoung.